srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -170,6 +170,7 @@ private static void addEventKey(String name, String symbol, int event, int flags
     addModifierKey("ordinal", "Ord", FLAG_ACCENT_ORDINAL | FLAG_SMALLER_FONT);
     addModifierKey("fn", "Fn", FLAG_FN | FLAG_SMALLER_FONT);
     addModifierKey("meta", "◆", FLAG_META);
+    addModifierKey("ctrl_alt", "C+A", FLAG_LOCK | FLAG_CTRL | FLAG_ALT | FLAG_SMALLER_FONT);
```

Comment:
Unfortunately, this is harder to implement now. See 36e10a7 and the "fake ptr" mechanism.